sqlite_utils/db.py

```diff
@@ -1271,12 +1288,34 @@ def init_spatialite(self, path: Optional[str] = None) -> bool:
         return result and bool(result[0])
 
 
+def _split_names(fullname: str) -> Tuple[str, str]:
+    if '.' not in fullname:
+        return '', fullname
```

Comment:
this is a choice to not make all the existing SQL more verbose by putting `main` everywhere - not only does this keep the SQL 'quieter', but it avoids having to refactor a tooon of tests that are asserting string values of SQL queries.